src/Domain/Creators/TimeDependentOptions/BinaryCompactObject.cpp

```diff
@@ -469,8 +488,14 @@ TimeDependentMapOptions<IsCylindrical>::distorted_to_inertial_map(
                                     : RotScaleTrans{};
 
   if (block_has_shape_map) {
-    if (rot_scale_trans_map_.has_value()) {
+    if (rot_scale_trans_map_.has_value() and
+        (use_rigid_map and skew_map_.has_value())) {
```

Comment:
If `use_rigid_map=false` but both `rot_scale_trans` and `skew` maps are given, this will ignore the skew map. Is that what you want? Could you make this clearer, maybe with a comment or by throwing errors or something

Reply:
Not "can't" be applied, but "won't" be applied. If there is a skew map, but the block we are in is in the outer shell, then `use_rigid_map=false` but `skew_map_.has_value()=true`.

Reply:
Ok that clarifies it, thanks 👍

I think in this code branch and also in the `block_has_shape_map` branch in `grid_to_distorted_map` below we're always within the inner cubes, so `use_rigid_map` should be always `true`. Maybe assert that instead of the sentinel.

Reply:
Added some errors